manager/snap.go

```diff
@@ -15,13 +15,11 @@ import (
 	"github.com/juju/proxy"
 )
 
-const (
-	// SnapExitCode is used to indicate a retryable failure for Snap.
+var (
+	// SnapExitCodes is used to indicate a retryable failure for Snap.
 	// See list of failures.
-	SnapExitCode = 1
-)
+	SnapExitCodes = []int{1, 2}
```

Comment:
Can you please put a comment here to clarify what each exit code signifies for the benefit of others that might be working on this part of the codebase in the future?